app/models/mixins/supports_feature_mixin.rb

```diff
@@ -78,18 +78,6 @@ def unsupported_reason(feature)
 
   # query the instance if the feature is supported or not
   def supports?(feature)
-    # AvailabilityMixin and that feature is used
-    method_name = "validate_#{feature}"
-    if respond_to?(:is_available?) && respond_to?(method_name)
-      # e.g.: {:available => true,  :message => nil}
-      rslts = send(method_name)
```

Comment:
@kbrock we'll need to convert all of these before we can remove this right?
```
app/models/vm_or_template/operations.rb:      return {:available => false, :message => unsupported_reason(:control)}
app/models/vm_or_template/operations.rb:    return {:available => true,   :message => nil}  if current_state.send(check_powered_on ? "==" : "!=", "on")
app/models/vm_or_template/operations.rb:    {:available => false,  :message => "The VM is#{" not" if check_powered_on} powered on"}
app/models/vm_or_template/operations.rb:    {:available => false, :message => "#{message_prefix} is not available for #{self.class.model_suffix} VM or Template."}
app/models/manageiq/providers/cloud_manager.rb:      {:available => true, :message => nil}
app/models/manageiq/providers/infra_manager.rb:      {:available => true, :message => nil}
app/models/manageiq/providers/physical_infra_manager.rb:      {:available => true, :message => nil}
app/models/manageiq/providers/container_manager.rb:      {:available => true, :message => nil}
app/models/manageiq/providers/container_manager.rb:      {:available => true, :message => nil}
app/models/manageiq/providers/container_manager.rb:      {:available => true, :message => nil}
app/models/manageiq/providers/cloud_manager/template.rb:    {:available => false,
app/models/host.rb:    {:available => true, :message => nil}
app/models/host.rb:    {:available => true, :message => nil}
```

There are probably more in plugins I haven't checked yet

Reply:
@agrare do we want to just hit these with a single followup PR?

Are we ok putting the supports into the middle of the trees?

e.g.:
```ruby
ContainerManager.supports :performance
```